source/win-preferences/schema/general.js

```diff
@@ -99,6 +99,13 @@ export default function () {
           }
         }
       ],
+      [
+        {
+          type: 'checkbox',
+          label: trans('dialog.preferences.sortFoldersFirst'),
```

Comment:
To make the translation string consistent with the other translations, please convert this to snake_case.